package.json

```diff
@@ -104,5 +104,11 @@
     "source-map-loader": "^4.0.0",
     "typescript": "^5.0.0",
     "webpack": "^5.40.0"
+  },
+  "dependencies": {
+    "feelers": "^0.1.0-alpha.4",
+    "preact-portal": "^1.1.3",
+    "react-draggable": "^4.4.5",
+    "react-resizable": "^3.0.4"
```

Comment:
Is any of this dependency needed, still? `feelers` should be installed from the properties panel and I think the other one is absolute without the popup.

Reply:
Nicely spotted, forgot to get rid